scipy/stats/_continuous_distns.py

```diff
@@ -4583,7 +4583,7 @@ def _entropy(self):
 
 
 class mielke_gen(rv_continuous):
-    r"""A Mielke's Beta-Kappa continuous random variable.
+    r"""Mielke's Beta-Kappa continuous random variable[1]_.
```

Comment:
this text shows up in the overview here: https://docs.scipy.org/doc/scipy/reference/stats.html

Better put the reference to the notes